pom.xml

```diff
@@ -32,6 +32,7 @@
     <!-- TODO: Ongoing work on https://github.com/MarkEWaite/git-plugin/tree/master-findbugs-fixes. -->
     <findbugs.failOnError>false</findbugs.failOnError>
     <workflow.version>1.14.2</workflow.version>
+    <scm-api-plugin.version>2.0.8-20170228.204551-2</scm-api-plugin.version> <!-- TODO https://github.com/jenkinsci/scm-api-plugin/pull/32 -->
```

Comment:
> I assume that interactive testing of all versions after this is merged

Well generally I would advise holding off on merge until the upstream is released, though it is at your discretion. `maven-release-plugin` will not allow this to be released until upstream is released.

> will need to mutate the version of the scm-api-plugin from the explicit version back the 2.0.8-SNAPSHOT

No that is not necessary.